pkg/commands/serviceresource/create.go

```diff
@@ -0,0 +1,120 @@
+package serviceresource
+
+import (
+	"io"
+
+	"github.com/fastly/cli/pkg/cmd"
+	fsterr "github.com/fastly/cli/pkg/errors"
+	"github.com/fastly/cli/pkg/global"
+	"github.com/fastly/cli/pkg/manifest"
+	"github.com/fastly/cli/pkg/text"
+	"github.com/fastly/go-fastly/v7/fastly"
+)
+
+// CreateCommand calls the Fastly API to create a resource link.
+type CreateCommand struct {
+	cmd.Base
+	jsonOutput
+
+	autoClone      cmd.OptionalAutoClone
+	input          fastly.CreateResourceInput
+	manifest       manifest.Data
+	serviceVersion cmd.OptionalServiceVersion
+}
+
+// NewCreateCommand returns a usable command registered under the parent.
+func NewCreateCommand(parent cmd.Registerer, globals *global.Data, data manifest.Data) *CreateCommand {
```

Comment:
FYI for consistency across the code base I've switched the argument names to `g` and `m`, for example:

```suggestion
func NewCreateCommand(parent cmd.Registerer, g *global.Data, m manifest.Data) *CreateCommand {
```

Reply:
Updated by [7b67275](https://github.com/fastly/cli/pull/800/commits/7b67275bf290b5eb4cfe0a7d7d8ea6a15165b441)